### PR TITLE
Made wishlist page

### PR DESCRIPTION
**Pull request by poojavishnoi**

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (anshtripathi01) [inline cart.html:5]: add author and description in meta tag

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+.cart_main_container{
+  min-width: fit-content;
+  display: flex;
+  flex-direction: row;
+}
+
+.price_details_container{
+  background-color: rgb(246, 240, 253);
```
Comment (anshtripathi01) [inline css/cart.css]: use hex-code for color

File: css/cart.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.cart_item .item_desc {
+  width: 30rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.secondary_outline_btn {
+  background-color: lightgrey;
```
Comment (anshtripathi01) [inline css/cart.css]: please keep above comment for color

File: cart.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+     <link
```
Comment (AtulHande03a) [inline cart.html:8]: add author and description tags

**Review by AtulHande03a — APPROVED** [review]

**Review by anshtripathi01 — APPROVED** [review]
it's not responsive for desktop.
Please do above changes
rest is good for me

File: cart.html (hunk 1 above)
Comment (poojavishnoi) [reply cart.html:5]: Okay.

File: css/cart.css (hunk 2 above)
Comment (poojavishnoi) [reply css/cart.css]: okay

File: cart.html (hunk 4 above)
Comment (poojavishnoi) [reply cart.html:8]: okay.

**Comment by poojavishnoi** [comment]
> it's not responsive for desktop. Please do above changes rest is good for me

I have made the changes, now I think its fine

**Pull request merged**